modules/claims_api/spec/factories/auto_established_claims.rb

```diff
@@ -241,7 +241,7 @@
     phase_chngd_dt { Faker::Time.backward(days: 5, period: :morning) }
     phase_type { 'Pending Decision Approval' }
     phase_type_change_ind { '76' }
-    ptcpnt_vet_id { Faker::Number.number(digits: 9) }
+    ptcpnt_vet_id { Faker::Number.number(digits: 9).to_s }
```

Comment:
This is a fix for a related test.  This value on a claim is a string so just want to make sure all the data is correctly set (I cannot speak to the other examples in here that do not do this and what that may mean for those tests).